python/lsst/pipe/tasks/healSparseMapping.py

```diff
@@ -684,3 +687,232 @@ def _compute_nside_coverage_tract(self, tract_info):
+
+        # Remove connections from those that are not configured
+        for name in BasePropertyMap.registry:
+            if name not in config.property_maps:
+                prop_config = BasePropertyMapConfig()
+                prop_config.do_min = False
+                prop_config.do_max = False
+                prop_config.do_mean = False
+                prop_config.do_weighted_mean = False
+                prop_config.do_sum = False
+            else:
+                prop_config = config.property_maps[name]
+
+            if not prop_config.do_min:
+                self.inputs.remove(f"{name}_map_min")
+                self.outputs.remove(f"{name}_consolidated_map_min")
+            if not prop_config.do_max:
+                self.inputs.remove(f"{name}_map_max")
+                self.outputs.remove(f"{name}_consolidated_map_max")
+            if not prop_config.do_mean:
+                self.inputs.remove(f"{name}_map_mean")
+                self.outputs.remove(f"{name}_consolidated_map_mean")
+            if not prop_config.do_weighted_mean:
+                self.inputs.remove(f"{name}_map_weighted_mean")
+                self.outputs.remove(f"{name}_consolidated_map_weighted_mean")
+            if not prop_config.do_sum:
+                self.inputs.remove(f"{name}_map_sum")
+                self.outputs.remove(f"{name}_consolidated_map_sum")
+
+
+class ConsolidateHealSparsePropertyMapConfig(pipeBase.PipelineTaskConfig,
+                                             pipelineConnections=ConsolidateHealSparsePropertyMapConnections):
+    """Configuration parameters for ConsolidateHealSparsePropertyMapTask"""
+    property_maps = BasePropertyMap.registry.makeField(
+        multi=True,
+        default=["exposure_time",
+                 "psf_size",
+                 "psf_e1",
+                 "psf_e2",
+                 "psf_maglim",
+                 "sky_noise",
+                 "sky_background",
+                 "dcr_dra",
+                 "dcr_ddec",
+                 "dcr_e1",
+                 "dcr_e2"],
+        doc="Property map computation objects",
+    )
+
+    def setDefaults(self):
+        self.property_maps["exposure_time"].do_sum = True
+        self.property_maps["psf_size"].do_weighted_mean = True
+        self.property_maps["psf_e1"].do_weighted_mean = True
+        self.property_maps["psf_e2"].do_weighted_mean = True
+        self.property_maps["psf_maglim"].do_weighted_mean = True
+        self.property_maps["sky_noise"].do_weighted_mean = True
+        self.property_maps["sky_background"].do_weighted_mean = True
+        self.property_maps["dcr_dra"].do_weighted_mean = True
+        self.property_maps["dcr_ddec"].do_weighted_mean = True
+        self.property_maps["dcr_e1"].do_weighted_mean = True
+        self.property_maps["dcr_e2"].do_weighted_mean = True
+
+
+class ConsolidateHealSparsePropertyMapTask(pipeBase.PipelineTask):
+    """Task to consolidate HealSparse property maps."""
```

Comment:
It would be helpful to document somewhere what exactly is being consolidated as it's not obvious to me.

Reply:
I am expanding this, and adding rst task documentation which I had forgotten to add.